### ISSUE TITLE
`Common mappers implementation

### ISSUE THREAD

**Issue by Siemienik** [Siemienik/xlsx-renderer#53]
Blocked by: #20

The PR #20 refactor how the code is organised. Among others there was created new layer called `mappers` (`src/mappers`) and here I will describe how I plan to expand it.

# Actual 

There is only file named `index.ts` with `MAPPER_DEFAULT` introduce in #19.

# Expected

This should contains handy and the most popular mappers ready to use. The first mappers how I consider to do is:

* lowerCaseMaper : `Some Text` => `some text` 
* upperCaseMaper: `Some Text` => `SOME TEXT`
* booleanMapper: `0` => `false`
* splitMapper: `val1,val2,val3` => `[val1,val2,val3]`
* jsonMapper : `{prop:[1,2]}` (string) => `{prop:[1,2]}` (object with array)
* numberMapper : `123` (string) => `123` (number)
* regexMapper - true if matches to regex 
* valueMatch - true if value match
* isEmpty - true if has no value
* isFilled - true if any value provided

For each mapper I'm going to create new issue.

# Request for a mapper

This issue is great a place to propose mappers, so if you have any idea, write a comment.
